requirements.txt

```diff
@@ -12,5 +12,6 @@ numpy
 pandas > 1.0
 pyproj > 2.2.0 
 pyyaml
+pywin32==224 ; sys_platform == 'win32'
```

Comment:
This is for compatibility with Emme, pywin32 gets pulled by one of the other dependencies, not sure which one, but setting the version fixes dll load issues. Specifying the sys_platform means this only applies on Windows, when installing on a different platform this will be ignored.